Gopkg.toml

```diff
@@ -62,42 +66,107 @@
 
 [[constraint]]
   name = "github.com/inconshreveable/go-update"
+  revision = "8152e7eb6ccf8679a64582a66b78519688d156ad"
 
 [[constraint]]
   name = "github.com/kardianos/osext"
 
 [[constraint]]
   name = "github.com/olekukonko/tablewriter"
-
-[[constraint]]
-  name = "github.com/pborman/uuid"
+  revision = "febf2d34b54a69ce7530036c7503b1c9fbfdf0bb"
 
 [[constraint]]
   name = "github.com/pkg/browser"
+  revision = "c90ca0c84f15f81c982e32665bffd8d7aac8f097"
 
 [[constraint]]
   name = "github.com/pkg/errors"
-  version = "0.8.0"
+  version = "=0.8.0"
 
 [[constraint]]
   name = "github.com/spf13/cobra"
+  revision = "e606913c4ee45fec232e67e70105fb6c866b95d9"
 
 [[constraint]]
   name = "github.com/spf13/pflag"
+  revision = "e57e3eeb33f795204c1ca35f56c44f83227c6e66"
 
 [[constraint]]
   name = "github.com/spf13/viper"
   revision = "382f87b929b84ce13e9c8a375a4b217f224e6c65"
 
-[[constraint]]
-  name = "golang.org/x/crypto"
-
 [[constraint]]
   name = "golang.org/x/oauth2"
+  revision = "442624c9ec9243441e83b374a9e22ac549b5c51d"
 
 [[constraint]]
   name = "gopkg.in/cheggaaa/pb.v1"
   version = "=1.0.13"
 
+# Get all dependencies for containers/image which is used for OCI image chaching
+[[constraint]]
+  name = "github.com/containers/image"
+  branch = "issue-368"
+  source = "https://github.com/hferentschik/image.git"
+
+[[constraint]]
+  name = "github.com/opencontainers/runc"
+  revision = "6b1d0e76f239ffb435445e5ae316d2676c07c6e3"
+
+[[constraint]]
+  name = "github.com/docker/go-connections"
+  revision = "3ede32e2033de7505e6500d6c868c2b9ed9f169d"
+
+# Use override for docker/docker and docker/distribution to ensure we get the specfied version.
+# In particular we need a version which uses the altest logrus
+[[override]]
+  name = "github.com/docker/docker"
+  revision = "30eb4d8cdc422b023d5f11f29a82ecb73554183b"
+
+[[override]]
+  name = "github.com/docker/distribution"
+  revision = "5f6282db7d65e6d72ad7c2cc66310724a57be716"
+
+[[constraint]]
+  name = "github.com/opencontainers/image-spec"
+  version = "=v1.0.0"
+
 [[constraint]]
   name = "gopkg.in/yaml.v2"
+  revision = "a3f3340b5840cee44f372bddb5880fcbc419b46a"
+
+[[constraint]]
+  name = "github.com/pborman/uuid"
+  revision = "1b00554d822231195d1babd97ff4a781231955c9"
+
+[[constraint]]
+  name = "golang.org/x/crypto"
+  revision = "453249f01cfeb54c3d549ddb75ff152ca243f9d8"
+
+[[constraint]]
+  name = "github.com/sirupsen/logrus"
+  version = "=v1.0.0"
```

Comment:
@hferentschik I do not think the changes are correct as per the individual commits in this PR. For example some changes including this one in Gopkg.toml should be part of https://github.com/minishift/minishift/pull/1685/commits/5138a0df75ad8dc63ac032d4abad1c67246fae71

Reply:
Same here , change related  to Upgrade docker-machine to 0.13.0 ?